tao_compiler/mlir/disc/transforms/disc_passes.td

```diff
@@ -368,3 +368,8 @@ def LhloFusionInlinerPass : Pass<"lhlo-fusion-inliner", "FuncOp"> {
     "after its body has been lowered";
   let constructor = "createLhloFusionInlinerPass()";
 }
+
+def DiscDotMergePass : Pass<"disc-dot-merge", "FuncOp"> {
+  let summary = "Dot merge optimization";
+  let constructor = "createDiscDotMergePass()";
```

Comment:
Better to have some explanation for what is "dot merge" here.

Reply:
I have updated the summary. But I have no idea why it is not shown here...